packages/react-magma-dom/src/components/TreeView/TreeView.test.js

```diff
@@ -330,6 +334,30 @@ describe('TreeView', () => {
       userEvent.click(getByTestId('item2-itemwrapper'));
       expect(onExpandedChange).not.toHaveBeenCalled();
     });
+
+    it('should work correctly with expandedAll() function', () => {
+      const onExpandedChange = jest.fn();
+      const apiRef = React.createRef();
+      const { getByText } = render(
+        <>
+          <button onClick={() => apiRef.current.expandAll()}>Expand All</button>
+          {getTreeItemsWithDisabled({
+            selectable: TreeViewSelectable.single,
+            initialExpandedItems: ['item1', 'item2'],
```

Comment:
This test has items 1 and 2 initially expanded, so it's not really testing that calling .expandAll() is expanding them